usaspending_api/accounts/models.py

```diff
@@ -89,49 +89,6 @@ def generate_tas_rendering_label(ATA, AID, TYPECODE, BPOA, EPOA, MAC, SUB):
         concatenated_tas = ATA + AID + TYPECODE + POAPHRASE + ACCTPHRASE
         return concatenated_tas
 
-    @staticmethod
```

Comment:
So great to remove all of this from the models--thank you for tackling this bit of tech debt as part of the story.